statistics-unit/src/main/java/com/dealflowbus/statisticsunit/models/Statistics.java

```diff
@@ -0,0 +1,25 @@
+package com.dealflowbus.statisticsunit.models;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Map;
+
+@Getter
+@Setter
```

Comment:
For data objects, you can use `@Data`. It's equivalent of getter , setter no arg constructor and generates hashcode and equals.

Reply:
It is because earlier i was applying toString methods to check flow of data, and with @Data i've had endless toString loops :)

Reply:
lombok won't generate it's toString if there is already one on class. No worries.

Reply:
But it can stay as it's now. It's not issue. Overflow of annotation is issue.